tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -458,6 +537,39 @@ private void prepareNativeRuntime() {
     }
   }
 
+
+  private void loadGraph(byte[] graphDef, Graph g) throws IOException {
+    final long startMs = System.currentTimeMillis();
+
+    if (VERSION.SDK_INT >= 18) {
+      Trace.beginSection("initializeTensorFlow");
+      Trace.beginSection("readGraphDef");
+    }
+
+
+    if (VERSION.SDK_INT >= 18) {
+      Trace.endSection(); // readGraphDef.
+      Trace.beginSection("importGraphDef");
+    }
+
+    try {
+      g.importGraphDef(graphDef);
+    } catch (IllegalArgumentException e) {
+      throw new IOException("Not a valid TensorFlow Graph serialization: " + e.getMessage());
+    }
+
+    if (VERSION.SDK_INT >= 18) {
+      Trace.endSection(); // importGraphDef.
+      Trace.endSection(); // initializeTensorFlow.
```

Comment:
This seems to match up with "readGraphDef", not "initializeTensorFlow".

In general the tracing is a good idea, but it should probably be added consistently to the entire interface in a separate PR.